ghub.el

```diff
@@ -138,6 +138,7 @@
          (url-request-data d))
     (with-current-buffer
         (url-retrieve-synchronously (concat ghub-base-url resource p))
+      (set-buffer-multibyte t)
```

Comment:
Doesn't Emacs [set buffers to multibyte by default](https://www.gnu.org/software/emacs/manual/html_node/elisp/Disabling-Multibyte.html)? What's causing the buffer to *not* be multibyte?